### ISSUE TITLE
Void Linux package template: wrong checksum, missing dependency

### ISSUE THREAD

**Issue by KeepBotting** [kozec/sc-controller#199]
~~The maintainter bumped the version number but neglected to update the checksum. It should be `774f59dd14ddd7b49f8a863f7dd731d4263f22d340e1f0b49a3cb647d5083b84`.~~

 ~~There is also a dependency missing, `kernel-libc-headers`. This package must be installed, otherwise an error is thrown and `sc-controller` will not start. [Reference](https://dev.solus-project.com/T1692).~~

I was able to get in touch with the maintainer. He has updated his template.

I have fixed these issues and sent a [pull request](https://github.com/voidlinux/void-packages/pull/6293) to the Void repository maintainers. Once the updated `sc-controller` is merged into the Void repository, I will update this repo's README and shoot @kozec a PR as well.

**Comment by kozec** [comment]
https://github.com/voidlinux/void-packages/pull/5952

**Comment by KeepBotting** [comment]
@kozec 

Yes, we're trying to get voidlinux/void-packages#5952 merged, I closed my PR in deference to 5952, didn't realize a PR for `sc-controller` was already present.

Void maintainers are dragging their feet, it's been ready to merge for months now but nobody has flipped the switch.

**Comment by kitling** [comment]
This was finally merged 11 days ago.

**Comment by KeepBotting** [comment]
Excellent. Opened #305 as promised.

**Comment by kozec** [comment]
Oh, glory hallelujah :D
